public/data/mehakagrawal.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "Mehak Agrawal",
+    "type": "personal",
+    "bio": "Software Engineer and Data Enthusiast",
+    "avatar": "https://github.com/mehakagrawal.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/mehakagrawal",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/mehakagrawall",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on LinkedIn",
+        "url": "https://www.linkedin.com/in/mehakagrawal/",
+        "icon": "linkedin"
+      },
+      {
+        "name": "Follow me on Medium",
+        "url": "https://medium.com/@mehakagrawal",
+        "icon": "medium"
+      },
+      {
+        "name": "Follow me on Kaggle",
+        "url": "https://www.kaggle.com/themehakagrawal",
+        "icon": "kaggle"
+      }
+    ]
+  }
```

Comment:
```suggestion
  }
  
```

Reply:
This will be taken care of by our GitHub actions.

Reply:
> This will be taken care of by our GitHub actions.

okay, that's nice